### PR TITLE
Linting on primary scada

### PR DESCRIPTION
**Pull request by preeth1**

### CODE REVIEW

**Comment by jessicamillar** [comment]
linting looks good. I can make a go at linting the remaining stuff before next time.

File: gw_spaceheat/actors/primary_scada/primary_scada.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
 from typing import List
-from actors.primary_scada.primary_scada_base import Primary_Scada_Base
+from actors.primary_scada.primary_scada_base import PrimaryScadaBase
 from data_classes.sh_node import ShNode
 from data_classes.boolean_actuator_component import BooleanActuatorComponent 
 from drivers.boolean_actuator.boolean_actuator_base import BooleanActuator
-from schema.gt.gt_telemetry.gt_telemetry_1_0_0_maker import  GtTelemetry100_Maker, GtTelemetry100
+from schema.gt.gt_telemetry.gt_telemetry_1_0_0_maker import GtTelemetry100
```
Comment (jessicamillar) [inline gw_spaceheat/actors/primary_scada/primary_scada.py:6]: We'll be making telemetry messages to send up to the AtomicTNode shortly, so this'll probably go back in soon.

Comment (preeth1) [reply gw_spaceheat/actors/primary_scada/primary_scada.py]: addressed here: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/1/commits/2ee2b70306b7b256ef6a5ee763441c2ef26990c5

**Review by jessicamillar — APPROVED** [review]
Looks good!

**Pull request merged**